matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/timeline/TimelineEventDecryptor.kt

```diff
@@ -38,23 +38,24 @@ internal class TimelineEventDecryptor(
     private val newSessionListener = object : NewSessionListener {
         override fun onNewSession(roomId: String?, senderKey: String, sessionId: String) {
             synchronized(unknownSessionsFailure) {
-                val toDecryptAgain = ArrayList<String>()
-                val eventIds = unknownSessionsFailure[sessionId]
-                if (eventIds != null) toDecryptAgain.addAll(eventIds)
-                if (toDecryptAgain.isNotEmpty()) {
-                    eventIds?.clear()
-                    toDecryptAgain.forEach {
-                        requestDecryption(it)
-                    }
-                }
+                unknownSessionsFailure[sessionId]
+                        .orEmpty()
+                        .toList()
```

Comment:
Reorder these two, `orEmpty` and `toList()`. So a new list isn't created when the list is null.

Reply:
ok, done